### ISSUE TITLE
[bitnami/ghost] Ghost madiadb PVC persistence size not applied

### ISSUE THREAD

**Issue by afermon** [bitnami/charts#2508]
**Which chart**:
bitnami/ghost:10.0.0

**Describe the bug**
I created a file values.yaml and set the parameters below to 3Gi. When installing the chart the PVC for mariadb was 8Gi.

```yaml
mariadb.persistence.size: 3Gi  # Readme
mariadb.master.persistence.size: 3Gi # Values.yaml
```

**To Reproduce**
1. Create a values.yaml file with the following content:
```yaml
ghostHost: "blog"
ghostUsername: "user@domain.com"
ghostPassword: "password"
ghostBlogTitle: "Blog"
allowEmptyPassword: false
persistence:
    size: 3Gi
mariadb:
    persistence:
        size: 3Gi # Readme 
    master:
        persistence:
            size: 3Gi # Values.yaml 
```
2. install the chart with:
```bash
helm install --values=./values.yaml xicre-blog bitnami/ghost
```
**Expected behavior**
mariadb PVC should have a 3Gi size

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.2.0", GitCommit:"e11b7ce3b12db2941e90399e874513fbd24bcb71", GitTreeState:"clean", GoVersion:"go1.13.10"}
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.2", GitCommit:"52c56ce7a8272c798dbc29846288d7cd9fbae032", GitTreeState:"clean", BuildDate:"2020-04-21T01:25:41Z", GoVersion:"go1.13.10", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.4+k3s1", GitCommit:"3eee8ac3a1cf0a216c8a660571329d4bda3bdf77", GitTreeState:"clean", BuildDate:"2020-03-25T16:13:25Z", GoVersion:"go1.13.8", Compiler:"gc", Platform:"linux/amd64"}
```

**Additional context**
Im using K3s

**Comment by afermon** [comment]
Found my issue,  existent PVCs size cannot be reduced, upgrade did not work. When I uninstalled the release it did not delete the original PVC and when I installed it again it took the same PVC that already existed, which had a 8Gi size from the first default install.   Manually deleted the PVC and reinstalled the chart and it now has the actual value.

`mariadb.master.persistence.size` is the correct key.

**Comment by afermon** [comment]
Found my issue,  existent PVCs size cannot be reduced, upgrade did not work. When I uninstalled the release it did not delete the original PVC and when I installed it again it took the same PVC that already existed, which had a 8Gi size from the first default install.   Manually deleted the PVC and reinstalled the chart and it now has the actual value.

`mariadb.master.persistence.size` is the correct key.
